<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cososys_qtClass</class>
 <widget class="QWidget" name="cososys_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cososys_qt</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextEdit" name="text_dictionary">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_dictionary">
      <property name="text">
       <string>Dictionary</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="text_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_text">
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="text_result">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_add_dictionary">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button_add_text">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="button_search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cososys_qt.qrc"/>
 </resources>
 <connections/>
</ui>
